src/text/span.rs

```diff
@@ -100,6 +120,78 @@ impl<'a> Span<'a> {
         }
     }
 
+    /// Builder method to set the content of the span.
+    ///
+    /// Accepts any type that can be converted to [`Cow<str>`] (e.g. `&str`, `String`, `&String`,
+    /// etc.).
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// # use ratatui::prelude::*;
+    /// let mut span = Span::default().content("content");
+    /// ```
+    #[must_use = "builder methods are lazy and do not modify the underlying object"]
```

Comment:
I like this! Never thought of using it on builder methods.

Reply:
Credit to @SOF3 in https://github.com/ratatui-org/ratatui/pull/600 for this

Reply:
For builders, there is no reason to construct a builder without using it anyway. Maybe better put `#[must_use]` on the `struct Builder` rather than its methods?